app/views/parents/_tab_materials.html.erb

```diff
@@ -0,0 +1,30 @@
+<script type="text/javascript">
+    $(document).ready( function() {
+        $.ajax({
+            type: 'POST',
```

Comment:
Forse una GET qui è concettualmente più corretta, ma se funziona non è un problema

Reply:
Sia in tab_assignments sia in parent che in teacher e in tab_marks in parent è sempre una post, quindi direi che le lasciamo cosi?